tests/fixtures/fake_config_file.txt

```diff
@@ -33,3 +33,5 @@ U http://fake.url-number.three/login?password=my_secret
 HosT https://fake.url-number.three
 DJ url-number.three
 
+T -HIDE hidden title
```

Comment:
I have never come across that in EZproxy... out of curiosity, do we use that often?

Reply:
not often  https://github.mit.edu/mitlibraries/Libproxy/blob/master/econtrol_config.txt#L119